### ISSUE TITLE
How to change the UI behaviour of survey

### ISSUE THREAD

**Issue by Charmi-2017** [surveyjs/surveyjs#621]
### Are you requesting a feature, reporting a bug or ask a question?

Question

### What is the current behavior?
Say there are 5 questions in a survey, I am on 3rd question now. I selected 9th answer for the 3rd question and click on next and went to the 4th question. Now I want to change my answer for Question 3, so I click on "Previous" button. When I go to 3rd question again the Marked answer shown is "1" by default, where as I had selected the 9th answer.

### What is the expected behavior?

If I go to previous question, it should show me the previously marked answer and that is "9" in this case

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
your_code_here

```

### Specify your

* browser:
* browser version:
* surveyjs platform (angular or react or jquery or knockout or vue):
* surveyjs version:

**Comment by andrewtelnov** [comment]
@Charmi-2017 Sounds like a bug to me. Could you provide us a plunker or fiddle example?

Thank you,
Andrew

**Comment by Charmi-2017** [comment]
Hi Andrew, 

Thanks for your quick response!
I am using Custom Widget - Nouislider.
Here is the url for an example - https://jsfiddle.net/surveyjs/a222qxpx/?utm_source=website&utm_medium=embed&utm_campaign=a222qxpx

There is only one Uislider here which is byfault set to "20-80". I want to keep byfeault as zero, but if the question is answered once, then it should save that and when the question is revisite, it should show the answer given by user and not the default one.
Thanks!

**Comment by andrewtelnov** [comment]
@Charmi-2017 Our Nouislider code doesn't work correctly, when there are several sliders in one survey. 
How urgent is this issue for you? 
We are going to rewrite the way we are presenting the custom widgets.
Initially the idea was to provide examples and show devs the way they may use third-party controls. However, the most people, as you, believes that it is our code and we have to support it.
So we decided to make the next step, improve the custom widget support and add them as plugins (optional npm packages) and start to support it fully.
If you can wait couple weeks, we likely create a new version of Nouislider and others.

Thank you,
Andrew

**Comment by Charmi-2017** [comment]
@andrewtelnov 
After diving into the code, I was able to solve the issue. Sorry to bother you for such a small fix. 
I have pushed the values in an array and extracting it from the array, when user revisits the question. So as of now its working as per our requirements.
Since in other examples, revisited questions showed up the answered values, I was bit unsure if I had done some mistake in the code.

Also, a Big Thank you to everyone involved for developing Survey related libraries and plugins as they really prove to be very helpful:)

Thank you,
Charmi
